src/DonorDashboards/resources/js/app/components/subscription-row/index.tsx

```diff
@@ -52,14 +52,14 @@ const SubscriptionRow = ({subscription}) => {
                         {__('View Subscription', 'give')} <FontAwesomeIcon icon="arrow-right" />
                     </Link>
                 </div>
-                {gateway.can_update && (
+                {(gateway.can_update || gateway.can_update_payment_method) && (
```

Comment:
Looks like we can make a const to be reused in this file like: 

```php
const gatewayCanUpdateSubscription = gateway.can_update || gateway.can_update_payment_method;
```

Reply:
Yeah, much clearer.